pom.xml

```diff
@@ -150,6 +151,7 @@
         <apache-rat-plugin.version>0.13</apache-rat-plugin.version>
         <ant-nodeps.version>1.8.1</ant-nodeps.version>
         <asm.version>7.1</asm.version>
+        <aws-sdk-version>1.11.562</aws-sdk-version>
```

Comment:
nit - Can you use `.version` like the other properties?

Reply:
Sure, will stick to the current nomenclature.